File: app/src/main/java/org/greenstand/android/TreeTracker/api/models/requests/AuthenticationRequest.java
package org.greenstand.android.TreeTracker.api.models.requests;

import com.google.gson.annotations.SerializedName;

/**
 * Created by zaven on 4/8/18.
 */

public class AuthenticationRequest {

    @SerializedName("client_id")
    private String clientId;

    @SerializedName("client_secret")
    private String clientSecret;

    public String getClientId() {
        return clientId;
    }

    public void setClientId(String clientId) {
        this.clientId = clientId;
    }

    public String getClientSecret() {
        return clientSecret;
    }

    public void setClientSecret(String clientSecret) {
        this.clientSecret = clientSecret;
    }
}

File: app/src/main/java/org/greenstand/android/TreeTracker/api/models/requests/AuthenticationRequest.kt
package org.greenstand.android.TreeTracker.api.models.requests

import com.google.gson.annotations.SerializedName
import org.greenstand.android.TreeTracker.BuildConfig

/**
 * Created by zaven on 4/8/18.
 */

data class AuthenticationRequest(@SerializedName("client_id")
                                 val clientId: String = BuildConfig.TREETRACKER_CLIENT_ID,
                                 @SerializedName("client_secret")
                                 val clientSecret: String = BuildConfig.TREETRACKER_CLIENT_SECRET,
                                 @SerializedName("device_android_id")
                                 val deviceAndroidId: String)
